web-animations/interfaces/AnimationEffectTiming/delay.html

```diff
@@ -58,5 +58,15 @@
   assert_equals(anim.effect.getComputedTiming().currentIteration, 0);
 }, 'Test finishing an animation using a large negative delay');
 
+test(function(t) {
+  var div = createDiv(t);
+  var anim = div.animate({});
+  for (let invalid of [NaN, Infinity]) {
+    assert_throws({ name: 'TypeError' }, function() {
+      anim.effect.timing.delay = invalid;
```

Comment:
Should this have an "animate() with invalid" test as well like iterationStart? What about the other properties included in the Blink test?